test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1368,6 +1368,12 @@ param([ValidatePattern(
             $res = TabExpansion2 -inputScript $TextInput -cursorColumn $TextInput.Length
             $completionText = $res.CompletionMatches.CompletionText
             $completionText -join ' ' | Should -BeExactly $ExpectedPropertyTypes
+
+            foreach ($match in $res.CompletionMatches) {
+                $completionText = $match.CompletionText.Replace("""", "").Replace("'", "")
+                $listItemText = $match.ListItemText
+                $completionText | Should -BeExactly $listItemText
+            }
```

Comment:
Would like to add a tool tip test here but I think last time we discussed it's not ideal because of localization.

Reply:
Yes.
Such a test is necessary, but it is only possible for a custom test completer.
I think we could do this later when we're done with the compiler interface enhancement - we'll need C# and script _feature_ tests.

Reply:
Yeah that's fine. I guess I am just wanting to have a test for every UX change including tool tips. Understand it is difficult and the more important parts to test is completion result anyways 😄.

Reply:
For example, for a specific completer, if possible values and tips are set in different places, then we could just check their presents and, maybe, number.